packages/grid/x-data-grid-pro/src/tests/clipboard.DataGridPro.test.tsx

```diff
@@ -48,13 +48,6 @@ describe('<DataGridPro /> - Clipboard', () => {
   describe('copySelectedRowsToClipboard', () => {
     let writeText;
 
-    before(function beforeHook() {
-      if (!isJSDOM) {
```

Comment:
Noticed that tests pass in real browser locally.
Maybe we shouldn't skip them anymore?

Reply:
Yeah, if they run with the real browser the `this.skip()` can be removed. The initial idea I had was to read the content in the clipboard but this needs to accept the permission request.